v8/rest/index.ts

```diff
@@ -393,6 +394,34 @@ export const Routes = {
 		return `/invites/${code}`;
 	},
 
+	/**
+	 * Route for:
+	 * - GET `/guilds/templates/{template.code}`
+	 * - POST `/guilds/templates/{template.code}`
+	 */
+	template(code: string) {
+		return `/guilds/templates/${code}`;
+	},
+
+	/**
+	 * Route for:
+	 * - GET `/guilds/{guild.id}/templates`
+	 * - POST `/guilds/{guild.id}/templates`
+	 */
+	guildTemplate(guildID: string) {
```

Comment:
This should probably be `guildTemplates`

Reply:
i documented it as template because you can actually only have one template per guild, it was changed before release so the routes are kinda inconsistent with the behaviour

Reply:
Still, naming usually follows its route counterpart 👁️